### PR TITLE
Add extra warning to 'Cluster-Aware Service Routing' page

### LINKED ISSUE #4317: Feedback on tasks/multicluster/split-horizon-eds page
Environment: Google Kubernetes Engine.
While running through this page, I had to switch on auto-scaling for cluster1 as pilot and telemetry pods didn't get scheduled because of insufficient memory and cpu. 

Should we mention this in the doc?

Autoscaling is not on by default in our cluster setup page in "docs/setup/kubernetes/platform-setup/gke/"

### PR DESCRIPTION
**Pull request by philrud**
Fixes #4317

### CODE REVIEW

File: content/docs/tasks/multicluster/split-horizon-eds/index.md [hunk 1]
```diff
@@ -49,6 +49,14 @@ This will be used to access pilot on `cluster1` securely using the ingress gatew
 
 1. Use Helm to create the Istio deployment YAML for `cluster1`:
 
+    {{< warning >}}
+    Enabling additional components necessary for multicluster operation may increase the resource footprint
+    of Istio control plane beyond the capacity of a default Kubernetes cluster created by following
```
Comment (frankbu) [inline content/docs/tasks/multicluster/split-horizon-eds/index.md]: of the Istio

File: content/docs/tasks/multicluster/split-horizon-eds/index.md [hunk 2]
```diff
@@ -49,6 +49,14 @@ This will be used to access pilot on `cluster1` securely using the ingress gatew
...
+    [Platform setup](/docs/setup/kubernetes/platform-setup/).
+    In case Istio components aren't getting scheduled due to insufficient CPU or memory consider
+    adding more nodes to your cluster or upgrade to instances of a larger size as necessary.
+    {{< /warning >}}
+
     {{< warning >}}
```
Comment (frankbu) [inline content/docs/tasks/multicluster/split-horizon-eds/index.md]: While you're at it, please remove this other warning about helm dependencies, which is no longer needed.

**Comment by philrud** [comment]
@yangminzhu linter is complaining on a couple of 404s
```
- ./public/zh/docs/tasks/security/authz-permissive/index.html
  *  External link https://raw.githubusercontent.com/istio/istio/master/samples/bookinfo/platform/kube/bookinfo-add-serviceaccount.yaml failed: 404 No error
- ./public/zh/docs/tasks/security/authz-tcp/index.html
  *  External link https://raw.githubusercontent.com/istio/istio/master/samples/bookinfo/platform/kube/rbac/ratings-v2-add-serviceaccount.yaml failed: 404 No error
```
looks like those were deleted as part of your commit:  https://github.com/istio/istio/commit/a87f2e018406b83e58bb4f55fe9d0e917881ba9d#diff-a70cbc3e5edf69197d1f24b779c1e25c could you please help updating corresponding docs pages accordingly? thanks!

File: content/docs/tasks/multicluster/split-horizon-eds/index.md (hunk 1 above)
Comment (philrud) [reply content/docs/tasks/multicluster/split-horizon-eds/index.md]: fixed

File: content/docs/tasks/multicluster/split-horizon-eds/index.md (hunk 2 above)
Comment (philrud) [reply content/docs/tasks/multicluster/split-horizon-eds/index.md]: done

**Comment by philrud** [comment]
besides the 2 linter failures mentioned above, there's another broken link -- apparently redis.io cert has just expired earlier today

File: content/docs/tasks/multicluster/split-horizon-eds/index.md [hunk 3]
```diff
@@ -50,9 +50,11 @@ This will be used to access pilot on `cluster1` securely using the ingress gatew
 1. Use Helm to create the Istio deployment YAML for `cluster1`:
 
     {{< warning >}}
-    If you're not sure if your `helm` dependencies are up to date, update them using the
-    command shown in [Helm installation steps](/docs/setup/kubernetes/install/helm/#installation-steps)
-    before running the following command.
+    Enabling additional components necessary for multicluster operation may increase the resource footprint
```
Comment (rcaballeromx) [inline content/docs/tasks/multicluster/split-horizon-eds/index.md]: Consider: 
    When you enable the additional components necessary for multicluster operation, the resource footprint
    of the Istio control plane may increase beyond the capacity of the default Kubernetes cluster you created when
    completing the [Platform setup](/docs/setup/kubernetes/platform-setup/) steps.
    If the Istio services aren't getting scheduled due to insufficient CPU or memory, consider
    adding more nodes to your cluster or upgrading to larger memory instances as necessary.

Comment (philrud) [reply content/docs/tasks/multicluster/split-horizon-eds/index.md]: updated as recommended

**Comment by philrud** [comment]
Could anyone please help merging this? This is part of 1.2 release qualification. Thanks!
Linter keeps failing due to unrelated issues that I unfortunately cannot address myself (see above).

cc @duderino @geeknoid

**Comment by rcaballeromx** [comment]
@philrud Please rebase the PR for the tests to pass.

**Comment by philrud** [comment]
@rcaballeromx done, looks like checks are passing now
@sdake thanks for taking care of those broken links

**Comment by philrud** [comment]
@rcaballeromx could you resolve your comments please? this can't be merged this until "the requested changed are addressed". thanks!

**Comment by rcaballeromx** [comment]
@philrud Approved! Thanks again for adding the warning.

**Pull request merged**